kernel/aster-nix/src/util/net/options/utils.rs

```diff
@@ -141,6 +142,30 @@ impl WriteToUser for LingerOption {
     }
 }
 
+impl ReadFromUser for timeval_t {
```

Comment:
`timeval_t` is a `Pod` type. So use `impl_read_write_for_pod_type` macro instead of implementing `ReadFromUser` and `WriteToUser` maunally.